internal/controller/standalone_pgadmin/configmap.go

```diff
@@ -181,3 +187,38 @@ func generateClusterConfig(
 	err := encoder.Encode(servers)
 	return buffer.String(), err
 }
+
+// generateGunicornConfig generates the config settings for the gunicorn server
+// - https://docs.gunicorn.org/en/latest/settings.html
+func generateGunicornConfig(pgadmin *v1beta1.PGAdmin) (string, error) {
+	settings := map[string]any{
+		// Bind to all IPv4 addresses and set 25 threads by default.
+		// - https://docs.gunicorn.org/en/latest/settings.html#bind
+		// - https://docs.gunicorn.org/en/latest/settings.html#threads
+		"bind":    "0.0.0.0:" + strconv.Itoa(pgAdminPort),
+		"threads": 25,
+	}
+
+	// Copy any specified settings over the defaults.
+	for k, v := range pgadmin.Spec.Config.Gunicorn {
+		settings[k] = v
+	}
+
+	// Write mandatory settings over any specified ones.
+	// - https://docs.gunicorn.org/en/latest/settings.html#workers
+	// - https://docs.gunicorn.org/en/latest/settings.html#workers
+	settings["workers"] = 1
+	settings["chdir"] = "/usr/local/lib/python3.11/site-packages/pgadmin4"
```

Comment:
@dsessler7 you and I spoke about this path the other day.

Reply:
❓ I see this path in a variable in the startup script. Is it possible to use that variable as a arg/flag to gunicorn?

Reply:
Yeah, that's an option as well. The command works out to be
`gunicorn -c /etc/pgadmin/gunicorn_config.py --chdir $PGADMIN_DIR pgAdmin4:app &`
This way is also a [more authoritative option](https://docs.gunicorn.org/en/latest/configure.html), so a value given through the spec should be overridden.
I'll make this change.